pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -1261,4 +1268,23 @@ public static class NullValuePlaceHolder {
     public static final byte[][] BYTES_ARRAY = new byte[0][];
     public static final Object MAP = Collections.emptyMap();
   }
+
+  public static class CursorConfigs {
+    public static final String DEFAULT_RESULT_STORE_TYPE = "file";
```

Comment:
I'm not sure if we should use this as a default. IICU this store type only works when queries go to the broker that opened it, which is difficult to control for users in the suggested deployments using k8s (unless there is only a single broker).

I understand this result store is useful during development, but honestly I would prefer to disable this feature by default and force users to provide an actual production ready result store

Reply:
`file` also supports HDFS and cloud store schemes.